modules/engine/test/transform/transform.spec.js

```diff
@@ -1350,7 +1350,7 @@ test('WebGL#Transform run (custom parameters)', t => {
   const outTexData = transform.getData({packed: true});
   t.deepEqual(outTexData, expectedData, `${name} Transform should write correct data into Texture`);
 
-  t.ok(getParameter(gl2, GL.BLEND) === true, 'Parameters are properly set');
+  t.ok(getParameters(gl2, [GL.BLEND])[GL.BLEND] === true, 'Parameters are properly set');
```

Comment:
`getParameters(gl, key)` -> `getParameters(gl, key)[key]`, this is breaking API change. Update `getParamters` doc and also upgrade guide?

Also why do we have to specify the `key` twice?

Reply:
It's `getParameter` (singular) to `getParameters` (plural) which returns an object. `getParameter` (singular) has been removed.